src/main/java/io/debezium/connector/vitess/pipeline/txmetadata/VitessEpochProvider.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.vitess.pipeline.txmetadata;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.debezium.connector.vitess.Vgtid;
+
+public class VitessEpochProvider {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(VitessEpochProvider.class);
+    private Map<String, Long> shardToEpoch = new HashMap<>();
+    private static final ObjectMapper MAPPER = new ObjectMapper();
+
+    public Long getEpochForGtid(Long previousEpoch, String previousGtidString, String gtidString) {
```

Comment:
This method doesn't use `shardToEpoch` instance variables, can be a static method

Reply:
Good catch, I'll update that one.